app/bundles/UserBundle/Entity/User.php

```diff
@@ -587,7 +585,7 @@ public function setRole(Role $role = null)
     /**
      * Get role.
      *
-     * @return Role
+     * @return ?Role
```

Comment:
Based on the metadata, there should not be a user without a `role`, why is this nullable?

Reply:
Reverted.